src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java

```diff
@@ -47,4 +49,10 @@ public SearchResultPage returnHomePage() {
         driver.findElement(By.xpath("//a/img")).click();
         return this;
     }
+
+	public ExternalPage goToResultLink (int numberResultLink) {
+		List<WebElement> searchResultLink = driver.findElements(By.xpath("//div[@class='yuRUbf']/a"));
```

Comment:
highly inefficient, use direct locator instead of getting the list and checking it's size

Reply:
(//a[text()='kitten'])[5]